arc/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Arc ClickBench Benchmark Runner
+# This script was used to generate the benchmark results
+
+TRIES=3
+PARQUET_FILE="hits.parquet"
+
+python3 << 'EOF'
+import duckdb
+import time
+import multiprocessing
+from pathlib import Path
+
+PARQUET_FILE = Path("hits.parquet")
+
+# Connect to DuckDB
+conn = duckdb.connect()
+
+# Configure optimizations
```

Comment:
Regarding l. 20-22: As per the benchmark rules, databases are supposed to run with their default settings. Especially since "arc/template.json" mentions `tuned: no`.

We will accept tuned configurations as well but we ideally do that as a follow-up PR.